tools/linker/MonoTouch.Tuner/RemoveBitcodeIncompatibleCodeStep.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2012-2014, 2016 Xamarin Inc. All rights reserved.
+using System.Collections.Generic;
+using Mono.Tuner;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Mono.Linker.Steps;
+
+using Xamarin.Bundler;
+using Xamarin.Linker;
+using Mono.Linker;
+using System;
+
+namespace MonoTouch.Tuner {
+	
+	public class RemoveBitcodeIncompatibleCodeStep : BaseStep {
```

Comment:
maybe a `SubStep` would be better (less code, since it shares the assembly/types/methods iterations) ?

Reply:
I've changed it to use a SubStep.